Cargo.toml

```diff
@@ -287,46 +287,49 @@ inherits    = "dev"
-sha2               = { default-features = false, version = "0.10" }
-thiserror          = { default-features = false, version = "2" }
-time               = { default-features = false, version = "0.3.44" }
-tokio              = { default-features = false, version = "1" }
-tokio-metrics      = { default-features = false, version = "0.4" }
-tonic              = { default-features = false, version = "0.14.1", features = ["tls-native-roots", "tls-ring"] }
-tonic-prost        = { default-features = false, version = "0.14.2" }
-tonic-prost-build  = { default-features = false, version = "0.14.1" }
-tracing            = { default-features = false, version = "0.1.41", features = ["attributes"] }
-tracing-log        = { default-features = false, version = "0.2" }
+async-trait = { default-features = false, version = "0.1" }
+base64 = { default-features = false, version = "0.22", features = ["std"] }
+bon = { default-features = false, version = "3" }
+bytes = { default-features = false, version = "1.10", features = ["serde"] }
+chrono = { default-features = false, version = "0.4.42" }
+crc32c = { default-features = false, version = "0.6.8" }
+futures = { default-features = false, version = "0.3" }
+http = { default-features = false, version = "1", features = ["std"] }
+http-body = { default-features = false, version = "1" }
+http-body-util = { default-features = false, version = "0.1.3" }
+hyper = { default-features = false, version = "1" }
+lazy_static = { default-features = false, version = "1" }
+percent-encoding = { default-features = false, version = "2" }
+pin-project = { default-features = false, version = "1" }
+prost = { default-features = false, version = "0.14", features = ["std"] }
+prost-build = { default-features = false, version = "0.14" }
+prost-types = { default-features = false, version = "0.14" }
+rand = { default-features = false, version = "0.9.2" }
+reqwest = { default-features = false, version = "0.12.23", features = ["json"] }
+rustls = { default-features = false, version = "0.23" }
+rustls-pemfile = { default-features = false, version = "2" }
+serde = { default-features = false, version = "1.0.226", features = ["serde_derive"] }
+serde_json = { default-features = false, version = "1.0.132", features = ["std"] }
+serde_with = { default-features = false, version = "3", features = ["base64", "macros", "std"] }
+sha2 = { default-features = false, version = "0.10" }
+thiserror = { default-features = false, version = "2" }
+time = { default-features = false, version = "0.3.44" }
+tokio = { default-features = false, version = "1" }
+tokio-metrics = { default-features = false, version = "0.4" }
+tonic = { default-features = false, version = "0.14.1", features = ["tls-native-roots", "tls-ring"] }
+tonic-prost = { default-features = false, version = "0.14.2" }
+tonic-prost-build = { default-features = false, version = "0.14.1" }
+tracing = { default-features = false, version = "0.1.41", features = ["attributes", "std"] }
```

Comment:
Do we need adding `std` here?  That means every client library gains that feature, even if they don't use it.  I suppose it is moot since `gax-internal` would be gaining it too?